distributed_loader.cc

```diff
@@ -330,9 +330,9 @@ future<> distributed_loader::open_sstable(distributed<database>& db, sstables::e
                     shards_interested_in_this_sstable.insert(shard_responsible_for_generation);
 
```

Comment:
subject should have susbsystem prefix, so people understand what scylla metadata is being not kept in memory.

Reply:
There is no way to attach a comment to the suject, so I attached it to the first line of the patch.

You can tell that I referred to the subject, by the fact that the word "subject" is used as the subject of the sentence.